ADRs/README.md

```diff
@@ -72,8 +72,10 @@ To contribute a new ADR, please follow these steps:
 3. Fill in the necessary information in the new ADR file, following the template structure.
 4. Use the large language model prompts, if desired, to generate or refine the ADR content.
 5. Submit a pull request to merge your ADR branch into the main branch, including a brief description of the problem and solution.
-6. Request a review from the team members, and address any feedback or suggestions provided.
-7. Once approved, merge the pull request to update the repository with the new ADR.
+6. Add a reference to the [ADR Index](https://www.notion.so/pocketnetwork/1a5893d2183b470b93610d2d9ce84d36?v=91b83d5b44d34ba6abf36f9070c2f3a8&pvs=4) for your pull request with appropriate status.
+7. Request a review from the team members, and address any feedback or suggestions provided.
+8. Once approved, merge the pull request to update the repository with the new ADR.
+9. If rejected or withdrawn, update the document's status as withdrawn, close the pull request, and update the status of the ADR in the [ADR Index](https://www.notion.so/pocketnetwork/1a5893d2183b470b93610d2d9ce84d36?v=91b83d5b44d34ba6abf36f9070c2f3a8&pvs=4).
```

Comment:
Do you think we need a "rejected" status? My thinking was that we might get away with just having "withdrawn" but I'm realizing that in the case where we have an unresponsive or irrational author, we would either have to update the status to withdrawn as the reviewer (which feels a little weird) or introduce a "rejected" status.

I intentionally avoided including a "rejected" status because of the negative connotation. After a bit of a back and forth with chatGPT, I had a list of candidate words and then looked at the definitions to find a best fit. "Unsuitable" rose to the top and with the exception of "discarded", I disqualified the rest:

1. **Unsuitable**: Not appropriate, fitting, or suitable for a particular purpose or situation; not matching the requirements or standards.
2. Discarded: To get rid of or throw away something that is no longer wanted, needed, or considered valuable.
1. ~~Neglected~~: To **pay little or no attention to, disregard, or fail to care** for someone or something, often resulting in their being **overlooked or not given proper consideration**.
3. ~~Forsaken~~: To abandon or desert someone or something, leaving them **alone, neglected, or without support or care**.
6. ~~Excluded~~: To **deliberately leave out** or **deny someone or something access, participation, or involvement** in a particular group, activity, or situation.
7. ~~Ignored~~: To deliberately **pay no attention to, disregard, or overlook** someone or something, often by choice or as a result of **indifference**.
8. ~~Abandoned~~: To completely **give up, desert, or forsake** someone or something, **leaving them without any support, care, or protection**.
9. ~~Dismissed~~: To reject, disregard, or wave aside someone or something **as unimportant, insignificant, or not worthy of consideration**.
10. ~~Disregarded~~: To intentionally **ignore, pay no attention to, or overlook** someone or something, often as a result of **neglect or indifference**.

TLDR; I think we should add an "unsuitable" state to handle the case of unresponsive and/or irrational contributors such that we can prevent ADRs from getting stuck in "in review" without requiring a second party (i.e. pokt) to "withdraw" them on behalf of said contributors (not the best look).

Reply:
I think unsuitable is a good replacement. I'll update the wording to match 👍